docs/cow-protocol/tutorials/cow-amm-deployer.mdx

```diff
@@ -2,82 +2,43 @@
 sidebar_position: 6
 ---
 
-# CoW AMM Deployer
-The [CoW AMM Deployer](https://app.safe.global/share/safe-app?appUrl=https%3A%2F%2Fdeploy-cow-amm.bleu.fi&chain=et) is a user-friendly Safe App that simplifies the process of deploying and managing [CoW AMMs](https://cow.fi/cow-amm) (Automated Market Makers) from Safe wallets. It enables users without programming skills to take advantage of the benefits of CoW AMM. 
+# Initial notes
 
-The app was developed by [bleu](https://github.com/bleu-fi) with sponsorship from the [CoW Grants DAO](https://grants.cow.fi/).
+The proposed transactions and interactions, can be executed through Safe's transaction builder, etherscan "Write Contract" or from other UI mechanisms or directly from console.
 
-This app is only accessible from a Safe wallet. 
+# Deploying a CoW AMM
 
-:::caution
+In this short tutorial, we describe how one can create and deploy their own Balancer CoW AMM.
+
+The current factory contract addresses are the following:
+- Ethereum Mainnet: [`0x23fcC2166F991B8946D195de53745E1b804C91B7`](https://etherscan.io/address/0x23fcC2166F991B8946D195de53745E1b804C91B7#code)
+- Gnosis Chain: [`0x7573B99BC09c11Dc0427fb9c6662bc603E008304`](https://gnosisscan.io/address/0x7573B99BC09c11Dc0427fb9c6662bc603E008304#code)
+- Sepolia Testnet: [`0x9F151748595bAA8829d44448Bb3181AD6b995E8e`](https://sepolia.etherscan.io/address/0x9F151748595bAA8829d44448Bb3181AD6b995E8e#code)
```

Comment:
If we are supplying addresses, we should likely also link to the repository where these addresses / deployment information are derived from so that readers can independently verify this.

Reply:
The repository to link is the following: https://github.com/balancer/cow-amm but the address need to be updated to:  

- Ethereum Mainnet: [`0xf76c421bAb7df8548604E60deCCcE50477C10462`](https://etherscan.io/address/0xf76c421bAb7df8548604E60deCCcE50477C10462#code)
- Gnosis Chain: [`0x703Bd8115E6F21a37BB5Df97f78614ca72Ad7624`](https://gnosisscan.io/address/0x703Bd8115E6F21a37BB5Df97f78614ca72Ad7624)#code)
- Sepolia Testnet: [`0x1E3D76AC2BB67a2D7e8395d3A624b30AA9056DF9`](https://sepolia.etherscan.io/address/0x1E3D76AC2BB67a2D7e8395d3A624b30AA9056DF9#code)